.github/workflows/release.yaml

```diff
@@ -0,0 +1,60 @@
+name: Build release image and push to hub
```

Comment:
the name will show in the action page, can we give an short one?

Reply:
Yes, we can! What would you suggest? `Release Image`?

Reply:
`Release Image`  is OK, you can merge it after revise.